lib/redis_client/connection_mixin.rb

```diff
@@ -39,7 +39,7 @@ def call(command, timeout)
       end
     end
 
-    def call_pipelined(commands, timeouts)
+    def call_pipelined(commands, timeouts, raise_exception = true)
```

Comment:
```suggestion
    def call_pipelined(commands, timeouts, exception: true)
```

I think a kw arg would make sense here too. That will require to rename the local variable to something else but no big deal.

Reply:
Yeah it's fine. It's a local variable, the name don't matter that much.

Reply:
Can even rename it to `first_error`.

Reply:
Will do!